dash/views/current_view.py

```diff
@@ -7,27 +7,55 @@
 				html.Div(
 					children = [
 						dcc.Graph(id='rpm_graph'),
-						html.Div(
-							html.H2('32RPM',id='rpm_text'),
-							className='bit'
-						)
 					],
-					id='rpm'
+					id='rpm_graph'
 				),
 				html.Div(
-					children = [	
+					children =[
 						html.Div(
-							html.H2('00:00:00',id='time_text'),
-							id='time',
-							className='bit'
+							children = [	
+								html.Div(
+									children = [
+										html.H2('DURATION ⏱️',id='time_text', className='bit_title'),
+										html.H2(id='time_value')
+									],
+									id='time',
+									className='bit'
+								),
+								html.Div(
+									children = [
+										html.H2('RESISTANCE 😤',id='resistance_text', className='bit_title'),
+										html.H2('',id='resistance_value')
+									],
+									id='resistance',
+									className='bit'
+								)
+							],
+							className="time_resistance"
 						),
 						html.Div(
-							html.H2('30',id='resistance_text'),
-							id='resistance',
-							className='bit'
-						)
+							children = [	
+								html.Div(
+									children = [
+										html.H2('RPM 🔄',id='rpm_text', className='bit_title'),
+										html.H2(id='rpm_value'),
+									],
+									id='rpm',
+									className='bit'
+								),
+								html.Div(
+									children = [
+										html.H2('HEART RATE 🫀',id='heart_rate_text', className='bit_title'),
```

Comment:
Maybe change to a more cartoon heart like ❤️? Personal preference at the end of the day so up to you!